### DISCUSSION TITLE
Add support for legend in RadarChart

### DISCUSSION THREAD

**Discussion by wrobbinz** [mantinedev/mantine#6403] [Feature requests]
When more than one instance is plotted on the radar chart, it can become difficult to distinguish the different values. A legend with the same `onHighlight` behavior as bar/scatter chart would help a user focus in on a specific instance.

Recharts radarchart with legend: https://recharts.org/en-US/examples/SpecifiedDomainRadarChart

**Comment by rtivital** [comment]
This feature will be available in 7.11.0 minor release
